src/qml/bitcoin.cpp

```diff
@@ -158,6 +160,188 @@ void setupChainQSettings(QGuiApplication* app, QString chain)
         app->setApplicationName(QAPP_APP_NAME_REGTEST);
     }
 }
+
+// added this function to set custom data directory when starting node
```

Comment:
Comments like that seem add no value. However, this sentence is more suitable for a commit message.

Reply:
Also removed. Thanks for pointing it out.